docs/illink-options.md

```diff
@@ -147,6 +147,12 @@ control options listed below.
 | Substition  | ILLink.Substitution.xml  |  --ignore-substitutions |
 | LinkAttributes  | ILLink.LinkAttributes.xml    |  --ignore-link-attributes |
 
+### Turning off warnings
+
+The `--nowarn` option prevents the linker from displaying one or more linker warnings by
+specifying its warning codes. All warning codes must be prepended with `IL` and multiple
+warnings should be separated with a comma or semicolon.
```

Comment:
Missing separated by spaces

Reply:
Although it's possible to separate warnings with spaces I think we shouldn't put that in the docs to favor using commas and semicolons.

Reply:
I also think that our command line parser will probably have trouble with spaces used that way...

Reply:
Sapces will only work if the warning list is quoted. Do you think we should remove the ability to use spaces/quote the warning list?

Reply:
I don't mind supporting it (especially if it's the same as csc)...